libscript/src/script-instance.cpp

```diff
@@ -2077,10 +2112,20 @@ bool MCScriptCallHandlerOfInstanceInternal(MCScriptInstanceRef self, MCScriptHan
                             t_signature = MCValueGetTypeInfo(t_handler);
                             
                             for(uindex_t i = 0; i < MCHandlerTypeInfoGetParameterCount(t_signature); i++)
-                                if (MCHandlerTypeInfoGetParameterMode(t_signature, i) != kMCHandlerTypeFieldModeIn)
-                                    MCScriptStoreToRegisterInFrameAndRelease(t_frame, t_arg_regs[i], t_linear_args[i]);
+                            {
+                                if (MCHandlerTypeInfoGetParameterMode(t_signature, i) == kMCHandlerTypeFieldModeIn)
+                                    continue;
+                                
+                                if (t_success)
```

Comment:
I've cleaned this up a bit - that particular loop is slightly different as it has to keep going even if one of the stores fails.